3p-ecosystem/src/main/java/com/google/homesampleapp/screens/commissionable/CommissionableFragment.kt

```diff
@@ -122,4 +145,100 @@ class CommissionableFragment : Fragment() {
       true
     }
   }
+
+  // -----------------------------------------------------------------------------------------------
+  // Composables
+
+  @Composable
+  private fun CommissionableRoute(commissionableViewModel: CommissionableViewModel) {
+    val beacons by commissionableViewModel.beaconsLiveData.observeAsState()
+    val beaconsList = beacons!!.toList()
+    CommissionableScreen(beaconsList)
+  }
+
+  @Composable
+  private fun CommissionableScreen(beaconsList: List<MatterBeacon>) {
+    LazyColumn(
+      Modifier.fillMaxSize()
+      // FIXME: How can we get autoscroll with LazyColumn?
+      // Statement below causes an error:
+      // "Vertically scrollable component was measured with an infinity maximum height constraints, which is disallowed."
+      //.verticalScroll(rememberScrollState())
```

Comment:
LazyColumn scrolls automatically, so you won't need the `.verticalScroll(rememberScrollState())` modifier. This modifier is used for layouts that don't have in built scrolling ability like `Column`, `Box` and `Row`.

The crash occurs because `Modifier.verticalScroll()` allows the layout to report an size larger than the constraints of the screen, and then scrolls that size. LazyColumn has an infinite size so the Modifier doesn't now what to do with it.

Reply:
So, how does one go about scrolling automatically to the end of the list when new items are added and the screen is full?